tests/TestCase/Datasource/PaginatorTest.php

```diff
@@ -26,6 +26,16 @@ class PaginatorTest extends TestCase
 {
     use PaginatorTestTrait;
 
+    /**
+     * fixtures property
+     *
+     * @var array
+     */
+    public $fixtures = [
```

Comment:
Hmm.. There fixtures weren't added to `SimplePaginatorTest` after removal from the trait.  Yet there are no test failures, which makes me think setting these fixtures is probably not required.

Reply:
Multiple tests fail if `$fixtures` is not added to PaginatorTest.